marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/mapping-functions/core.sjs

```diff
@@ -137,24 +138,33 @@ function parseDate(value, pattern) {
     }
   }
   else {
-    fn.error(null, "The given date pattern (" + pattern + ") is not supported.");
+    try {
```

Comment:
Should we still keep the earlier error message for the case when pattern is not supported.
For ex, if "Mon YYYY DD" is not supported and the value is "Jan 2019 16", the error message would say
`The pattern 'Mon YYYY DD' cannot be applied to the value 'Jan 2019 16'`

The value seems to match the pattern and we should probably say it is not supported(?)

Reply:
I think that makes sense for the 3 non-standard formats.